orchestration/scripts/extract-json-value.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+import argparse
+import json
+
+# Argument parsing
+parser = argparse.ArgumentParser()
+parser.add_argument('--file', type=str, required=True, help='The file where the JSON is stored')
+parser.add_argument('--key', type=str, required=True, help='The key used to pull a value from the JSON data')
+args = parser.parse_args()
```

Comment:
I'm not sure argparse will work well for us here. At runtime, Argo will effectively be running `python <(cat source)`, so there's no real room to pass in command-line args.

The [existing scripts](https://github.com/DataBiosphere/clinvar-ingest/tree/master/orchestration/scripts) work around this by assuming some environment variables will be set by Argo. We could do the same here, and instead of pulling from `args` we pull from `os.environ`.

Reply:
Removing this script entirely in my next commit.